### DISCUSSION TITLE
video demo of the plugin? need help to understand

### DISCUSSION THREAD

**Discussion by Rhydderch** [aidenlx/obsidian-zotero#126] [General]
Hello,

I like how smooth the use of the plugin is. Creating and updating literature notes is seamless.
Thank you so much for creating it!

However, I don't understand how to put to good use the plugin.

Could you make a video? and perhaps a summary of which features listed here are implemented vs not yet implemented?

**Comment by aidenlx** [comment]
I'll finish the document and introduction before submission for plugin review. For now you can take a look at [the demo here](https://forum.obsidian.md/t/pdf-zotero-obsidian-current-state-and-collaboration-for-the-one-plugin/34372/47?u=aidenlx), although it may be a bit outdated.

**Reply by Rhydderch** [reply]
Great, and thank you for the link to the demo.

**Comment by aidenlx** [comment]
New document is available in obzt.aidenlx.top, make sure to check out latest version and updated tutorial!
